src/userSurvey.js

```diff
@@ -38,7 +38,10 @@ import popupNoPromptPage from "./html/userSurvey.popupNoPrompt.html";
 
 permissions.check({
     module: "webScience.userSurvey",
-    requiredPermissions: [ "notifications", "webRequest" ]
+    requiredPermissions: [ "notifications", "webRequest" ],
+    requiredContentSecurityPolicy: {
+        "script-src": [ "'sha256-7MkXA5Z7wxRyznhUZN3nVVs9GEQpvRXdYihZZqR2y6w='", "'sha256-l+kgCjP15GJlVSDL9qMNffrHu8mxJcag42o2TYofOUM='" ]
```

Comment:
Is this the hash for `userSurvey.popupNoPrompt.html`, does that mean we need to update it every time here? Seems like something we could do at build time with the rollup `replace` plugin.

Reply:
One of the hashes is for the inline script in `userSurvey.popupNoPrompt.html`, and the other is for the inline script in `userSurvey.popupPrompt.html`.  We can change the HTML and CSS in those files without updating the hashes; only changes to the inline JS will require new hashes. While I would like us to dynamically generate the hashes eventually, I also want them to be as stable as possible... a hash change is effectively a breaking API change for library users.